packages/launch/src/hooks/use-domain-selection.ts

```diff
@@ -42,15 +42,10 @@ export function useDomainSuggestionFromCart(): DomainSuggestions.DomainSuggestio
 
 	const domainName = domainProductFromCart?.meta;
 
-	const domainSuggestion = useSelect( ( select ) =>
-		domainName
-			? select( DOMAIN_SUGGESTIONS_STORE ).getDomainSuggestions( domainName, {
-					quantity: 1,
-					include_wordpressdotcom: false,
-					include_dotblogsubdomain: false,
-			  } )?.[ 0 ]
-			: undefined
-	);
+	const domainSuggestion = useDomainSuggestions( domainName, 1, undefined, undefined, {
+		include_wordpressdotcom: false,
+		include_dotblogsubdomain: true,
```

Comment:
cc @razvanpapadopol for knowledge, since this was recently discussed in a p2

Reply:
@ciampo, are you referring to pbAok1-1MG-p2#comment-3587 ? 
I believe we need to do more than just turn this flag to `true` in order to achieve what was mentioned there. #41219 is a dedicated GH issue for the explicit matching of TLDs and .blog subdomains.

Reply:
Yes, I was referring to that. I don't have as much perspective as you do on the matter, happy to track this in a separate issue.